core/drand_control.go

```diff
@@ -588,13 +588,14 @@ func (d *Drand) Status(c context.Context, in *drand.StatusRequest) (*drand.Statu
 	// Reshare status
 	switch {
 	case !d.dkgDone:
-		reshareStatus.Status = uint32(ReshareInProgress)
-	case d.dkgDone && d.receiver != nil:
 		reshareStatus.Status = uint32(ReshareNotInProgress)
+	case d.dkgDone && d.receiver != nil:
+		reshareStatus.Status = uint32(ReshareInProgress)
 	}
```

Comment:
did we have this logic backwards before?
if `!d.dkgDone` (the first case), why is the status `ReshareNotInProgress` now?